package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "scripture-burrito",
+    "version": "0.2.0",
+    "description": "Schema support for Scripture Burrito",
+    "main": "schema/index.js",
+    "directories": {
+        "doc": "docs"
+    },
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
This should probably reflect our internal test suite with AJV, this bit: https://github.com/bible-technology/scripture-burrito/blob/develop/.github/workflows/main.yml#L16